src/model/Column.ts

```diff
@@ -141,15 +154,22 @@ export default class Column extends AEventDispatcher {
    * @private
    */
   private compressed = false;
-  renderername: string;
-  protected rendererList: {type: string, label: string}[];
+
+
+  protected rendererInfo: IRendererInfo;
```

Comment:
while structure of fields in good in general. In this case, it is has a side effect that you don't wanna have. Since the field is protected sub classes can write to them. While they should be able to write to `rendererList`. They should use the setter to set the `rendererType` otherwise the changes are propagated.